src/shared/components/search-result/search-result.component.html

```diff
@@ -14,7 +14,8 @@ <h2 class="heading-h2">{{ caseState ? 'Case List' : 'Search result' }}</h2>
         <tr>
           <div class="search-result-column-label" (click)="sort(col)">{{col.label}}</div>
           <div *ngIf="comparator(col)" class="search-result-column-sort">
-            <a (click)="sort(col)" class="sort-widget" [innerHTML]="sortWidget(col)" href="javascript:void(0)"></a>
+            <a (click)="sort(col)" class="sort-widget" [innerHTML]="sortWidget(col)" href="javascript:void(0)"
+               attr.aria-label="Order by {{ isSortAscending(col)? 'Ascending' : 'Descending' }} {{col.label}}"></a>
```

Comment:
maybe shall we lower case Ascending and Descending?

Reply:
Done.